tests/e2e/specs/conference.spec.js

```diff
@@ -1,24 +1,21 @@
-// about cypress and iframes https://www.cypress.io/blog/2020/02/12/working-with-iframes-in-cypress/
-const getIframeDocument = () => cy
-  .get('#jitsiConferenceFrame0')
-  .its('0.contentDocument')
-  .should('exist');
-
-const getIframeBody = () => getIframeDocument()
-  .its('body').should('not.be.undefined')
-  .then(cy.wrap);
-
 describe('Conference.vue', () => {
+  const ROOM = 'SOME_ROOM_NAME';
+  const isValidName = ROOM.match(/^[^?&:"'%#]+$/);
```

Comment:
The name can be renamed. Also added jitsi create room check pattern. Can be used to verify if the name is valid.

Reply:
Yes, this is a good one.